lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2134,6 +2134,78 @@ static void optimizeQuantization(Function *F) {
   optimizeQuantizedMaxSplat(F);
 }
 
+#define BOILER_PLATE_TEMPLATE(NODE_NAME_)                                      \
```

Comment:
I think BOILER_PLATE_TEMPLATE is more confusing than the code it replaces.  I'd honestly rather see the substituted code (esp if `boilerPlateResolve` is turned into something descriptive).